src/pages/products/edit.js

```diff
@@ -0,0 +1,301 @@
+import React, { useState } from 'react';
+import { Form, Input, Button, notification, Select } from 'antd';
+import { useParams } from 'react-router-dom';
+
+const { Option } = Select;
+
+const formItemLayout = {
+  labelCol: { span: 6 },
+  wrapperCol: { span: 14 },
+};
+
+const ProductEdit = (props) => {
+  const [tags, setTags] = useState([]);
+  const [categories, setCategories] = useState([]);
+  const [currencies, setCurrencies] = useState([]);
+  const [productType, setProductType] = useState([]);
+  const [product, setProduct] = useState();
+
+  const { id } = useParams();
+  React.useEffect(() => {
+    fetch(process.env.REACT_APP_API_URL + `/products/${id}`)
+      .then((data) => data.json())
+      .then((data) => {
+        setProduct(data);
+      })
+      .catch((err) => {
+        notification.error({
+          message: 'Error',
+          description: 'Something went wrong',
+        });
+      });
+    fetch(process.env.REACT_APP_API_URL + '/tags')
+      .then((data) => data.json())
+      .then((data) => {
+        setTags(data.nodes);
+      });
+    fetch(process.env.REACT_APP_API_URL + '/categories')
+      .then((data) => data.json())
+      .then((data) => {
+        setCategories(data.nodes);
+      });
+    fetch(process.env.REACT_APP_API_URL + '/currencies')
+      .then((data) => data.json())
+      .then((data) => {
+        setCurrencies(data.nodes);
+      });
+    fetch(process.env.REACT_APP_API_URL + '/types')
+      .then((data) => data.json())
+      .then((data) => {
+        setProductType(data.nodes);
+      });
+  }, []);
```

Comment:
React Hook React.useEffect has a missing dependency: 'id'`
```suggestion
  }, [id]);
```